README.md

```diff
@@ -17,24 +17,29 @@ This plugin is in Beta. Live support is available for the latest builds through
 - [X] Support multiple island designs via Sponge schematics
 - [X] Expanding islands using claim blocks
 - [X] Isolation of islands to their own Minecraft region file
-- [X] Allow spawn/tp on your island at a configurable location 
-- [X] Use GriefPrevention trust commands to work in teams
+- [X] Allow spawn/tp on your island at a configurable location
```

Comment:
@subtank would you be able to translate these readme changes?

Reply:
yes.i am working on this.

Reply:
@Mohron working on chinese change log.